platform/platform-app/src/main/resources/features.xml

```diff
@@ -981,6 +981,7 @@
         <feature prerequisite="true">platform-api</feature>
         <feature prerequisite="true">platform-security-session</feature>
         <feature prerequisite="true">platform-migrate-all</feature>
+        <bundle>mvn:ddf.platform/platform-logging/${project.version}</bundle>
```

Comment:
Should we have a separate feature for this and put it right after `platform-api`? I'd think we want this to come up before anything else so we don't miss any logs.